presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -559,6 +559,14 @@ Procedures
 
     Create an empty partition in the specified table.
 
+* ``system.sync_partition_metadata(schema_name, table_name, mode)``
+
+    Check and repair partitions in metastore. There are three modes available:
```

Comment:
Can we get rid of "repair"?

```suggestion
    Check and update partitions list in Metastore. There are three modes available:
```

Reply:
We lowercase "metastore" elsewhere in the documentation